modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowAssetPublisher.testcase

```diff
@@ -232,4 +232,76 @@ definition {
 			webContentTitle = "Blogs Entry Title");
 	}
 
+	@description = "This is a test for LRQA-69814. Add a basic web content with workflow via asset publisher and approve it. "
+    @ignore = "Test Stub"
+    @priority = "3"
+    test AssetPublisherApproveWC {
+
+         // TODO LRQA-69814 AssetPublisherApproveWC pending implementation
+
+    }
+
+    @description = "This is a test for LRQA-69815. Add a basic web content with workflow via asset publisher and reject it. "
+    @ignore = "Test Stub"
+    @priority = "3"
+    test AssetPublisherRejectWC {
+
+        // TODO LRQA-69815 AssetPublisherRejectWC pending implementation
+
+    }
+
+    @description = "This is a test for LRQA-69816. Add a blog with workflow via asset publisher and approve it. "
+    @ignore = "Test Stub"
+    @priority = "3"
+    test AssetPublisherApproveBlog {
+
+        // TODO LRQA-69816 AssetPublisherApproveBlog pending implementation
+
+    }
+
+    @description = "This is a test for LRQA-69817. Add a blog with workflow via asset publisher and reject it. "
+    @ignore = "Test Stub"
+    @priority = "3"
+    test AssetPublisherRejectBlog {
+
+        // TODO LRQA-69817 AssetPublisherRejectBlog pending implementation
+
+    }
+
+    @description = "This is a test for LRQA-69818. Add a calendar event with workflow via asset publisher and approve it. "
+    @ignore = "Test Stub"
+    @priority = "3"
+    test AssetPublisherApproveCalendarEvent {
+
+        // TODO LRQA-69818 AssetPublisherApproveCalendarEvent pending implementation
+
+    }
+
+    @description = "This is a test for LRQA-69819. Add a calendar event with workflow via asset publisher and reject it. "
+    @ignore = "Test Stub"
+    @priority = "3"
+    test AssetPublisherRejectCalendarEvent {
+
+        // TODO LRQA-69819 AssetPublisherRejectCalendarEvent pending implementation
+
+    }
+
+    @description = "This is a test for LRQA-69820. Add a knowledge base article with workflow via asset publisher and approve it. "
+    @ignore = "Test Stub"
+    @priority = "3"
+    test AssetPublisherApproveKnowledgeBaseArticle {
+
+        // TODO LRQA-69820 AssetPublisherApproveKnowledgeBaseArticle pending implementation
```

Comment:
Hi Kristin! Could you, please, remove this stub? Thank you! :)

Reply:
Removed stub!